wp-content/themes/access/single-programs.php

```diff
@@ -46,6 +46,71 @@
 
 $context['post'] = $program;
 
+$context['schema'] = [
+  array(
+    "@context" => "https://schema.org",
+    "@type" => "GovernmentService",
+    "name" => $program->name,
+    "alternateName" => $program->plain_language_program_name,
+    "datePosted" => $program->post_modified,
+    "expires" => $program->custom['program_status_clear_date'],
```

Comment:
The program status clear date only applies to the special announcement object below so it could be moved there. Also, it will most likely be null in most cases. Perhaps do not set it if null

Reply:
@devowhippit thank you for your feedback, I removed the expires field and added the conditional to the `SpecialAnnouncement`